spec/lib/dspace/dspace_spec.rb

```diff
@@ -0,0 +1,157 @@
+# NOTE: Error-prone, and rspec makes testing against DSpace difficult, skipping 
+#   testing for now
+
+describe DSpace do
+
+  describe '.load' do
+    before do 
+      # TODO: Reset DSpace before each load
+      #   DSpace.context.abort is a good start, but variables need to be reset as well
+    end
+
+    after :all do
+      # TODO: Make sure DSpace is loaded properly for all tests
+    end
+
+=begin
```

Comment:
While this is syntactically valid, this is a fault of mine given that I don't have RuboCop enforcing certain style rules. If I recall correctly, DRDS uses a style rule which prefers blocks of comments to this.

Reply:
Fixed in latest commit. Should I open an issue to setup Rubocop in `dspace-cli` and `dspace-jruby`?